content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx

```diff
@@ -153,83 +161,142 @@ The next step is to sign up for Prisma's [Data Proxy](/data-platform/data-proxy)
+### Result
 
-![Prisma Data Browser](./450-06-data-browser.png)
+This confirms that your application works when you run it from your local environment.
 
-It's working locally!
+## 8. Create a repository and push to GitHub
+
+Create a private repository for your application on GitHub. You need the GitHub repository to add your project to Deno Deploy and enable automated deployments on push.
+
+<Admonition>
+
+**Note**<br /><br />
+
+Optionally, you can also [link your GitHub repository to your project](/data-platform/environments/create#projects-with-an-introspected-schema-or-with-no-schema) in the Prisma Data Platform so that you can keep it up-to-date with the latest changes you make in the `schema.prisma` file.
+
+</Admonition>
+
+### Steps
+
+1. [Create a private GitHub repository](https://github.com/new).
+  ![Create a private repository](./550-01-create-repo.png)
+2. Initialize your repository locally and push your changes to GitHub.
+  ```terminal
+  git init -b main
+  git remote add origin https://github.com/<username>/prisma-deno-deploy
+  git add .
+  git commit -m "initial commit"
+  git push -u origin main
+  ```
 
 ## 9. Deploy to Deno Deploy
 
-Finally we are ready to also deploy our tiny Deno application to Deno Deploy:
+You can now add and deploy your application to Deno Deploy.
+
+### Steps
+
+1. Go to [https://dash.deno.com/new](https://dash.deno.com/new).
+1. Select a GitHub organization or user and then select a repository.
+1. Select a production branch and select **Automatic** mode so that Deno Deploy can deploy every time you push a change to the repository.
+1. Select `index.ts` as the entry point to your project.
+1. To define the Data Proxy connection string, click **Add Env Variable**.
+    1. For **KEY**, enter `DATABASE_URL`.
+    1. For **VALUE**, paste the Data Proxy connection string.
+    ![Deno Deploy - project parameters](images/550-0X-deploy-to-deno-deno-project-parameters.png)
+1. Click **Link**.<br />
+  Wait for the first Deno deployment to finish.
+
+### Result
+
+When the first deployment finishes, your browser is redirected to the project view.
+
+### What's next
 
-1. [https://dash.deno.com/new](https://dash.deno.com/new)
-1. Select your GitHub organization or user, repository and branch
-1. Select the "Automatic" deployment mode
-1. Select `index.ts` as an entry point
-1. Make sure that the project has a good name
-1. "Add Env Variable" to add `DATABASE_URL` with the Data Proxy connection string
-1. Click "Link"
-1. Wait for the deployment to finish.
-   When it finishes, it redirects you to a project view.
-1. Click the blue "View" button at the top right to open your Deno Deploy deployment.
-   The browser window shows the same as locally.
+Click the blue **View** button at the top right to open the deployed Deno application.
 
-Visit your deployment URL. You'll see the following:
+If it works correctly, you can see the same result as when you tested locally.
```

Comment:
```suggestion
If it works correctly, you see the same result as when you tested locally.
```

Reply:
I am also not sure if we should include things like "if it works correctly" in our docs. We should project confidence that our products work as described. For a guide though, I'm less certain.

Reply:
I've reworded this to remove the conditional. It now says: "The application shows a similar result as when you tested locally".